routing/cross_routing_context.hpp

```diff
@@ -70,6 +70,8 @@ struct OutgoingCrossNode
   void Save(Writer & w) const;
 
   size_t Load(Reader const & r, size_t pos, size_t adjacencyIndex);
+
+  m2::RectD const GetLimitRect() const { return m2::RectD(m_point.lat, m_point.lon, m_point.lat, m_point.lon); }
```

Comment:
NoNo, I'm not asking you to do that right now, but keep in mind that m4::Tree should work with planes, not spheres.